crdppf/lib/wmts_parsing.py

```diff
@@ -0,0 +1,142 @@
+# -*- coding: UTF-8 -*-
+
+from xml.dom.minidom import parseString
+
```

Comment:
I like to make thing clear when importing stuff, like making a difference between SQLAlchemy stuff and other Python libs...

I will review the whole thing to be sure not to have miss a thing...